crates/solidity/testing/perf/npm/src/solc.typed.ast.runner.mts

```diff
@@ -0,0 +1,38 @@
+import { CompilationOutput, CompileFailedError, CompilerKind, compileSol } from "solc-typed-ast";
```

Comment:
This package has problems with its exported types, according to tsc. This is the reason why CI is failing right now. We can either drop it, or find an alternative to ignore this error.